### DISCUSSION TITLE
Kudos

### DISCUSSION THREAD

**Discussion by hairlesshobo** [photoprism/photoprism#2065] [General]
I just wanted to take a second and say that I am super impressed with what you have built so far. I have been looking for a self-hosted Google photos replacement that my wife approves of for over 2 years and nothing came close. About 9 months ago I tried PhotoPrism and really liked it, but it was missing the face tagging, which is crucial for my wife and I. I was very thrilled and impressed to see that you now have face tagging and in general, the app is coming along very well. Last night I finally "pulled the trigger" and pointed Prism at my library of ~28K photos from our phones over the past 14 years. Performance is fantastic, especially considering that I am running it on a 4th generation i3. Seeing what you have accomplished thus far, your excellent documentation, what you have planned in the roadmap, and your stance on "no bugs" are all the reasons I felt compelled to sponsor this project. As a life-long developer myself, I know just how much work goes into a project like this and you are doing a fantastic job making it feel very polished and professional, which too many projects lack.

In short, thank you both for all your hard work.

**Comment by lastzero** [comment]
That's our favorite kind of feedback :)

Note that MariaDB settings should be adjusted for large libraries. If you ever have problems, see the Performance Tips section in our docs:

https://docs.photoprism.app/getting-started/troubleshooting/performance/

We started developing PhotoPrism for our own needs, but soon realized that there are many other people waiting for something similar. Right now, the workload is too high for two people - so a top priority is to improve funding and then grow our team.

Thanks a lot for supporting our work!

Michael
